src/admin/sections/TokenGen.module.scss

```diff
@@ -0,0 +1,34 @@
+.tokenNumberInput,
+.button {
+  border: none;
+  background-color: var(--input-background);
+  color: var(--input-color);
+  padding: 5px 10px;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: 1rem;
+  text-decoration: none;
+
+  & + .tokenNumberInput,
+  & + .button {
+    margin-left: 1em;
+  }
+}
+
+.tokenNumberInput {
+  width: 4rem;
+  margin-left: 1em;
+  margin-right: 1em;
+  text-align: center;
+}
+
+.tokenList {
+  list-style-type: none;
+  margin: 1rem 0;
+  padding: 0;
```

Comment:
Maybe give tokenList a `max-height` and `overflow-y: auto`. Otherwise when creating more than 10 tokens, it becomes tedious to scroll down to either download or clear the tokens.
Might also add a different background to it like in the `Create new Ballot` form.